packages/react-ui-core/src/Menu/Menu.js

```diff
@@ -93,28 +95,26 @@ export default class Menu extends PureComponent {
     }
   }
 
-  get value() {
-    return this.options[this.state.highlightIndex]
-  }
-
   get options() {
     return this.props.options || []
   }
 
   get items() {
-    return this.options.map(option => (
-      typeof option === 'object' ? option.label : option
-    ))
+    return this.options.map(option => ((typeof option === 'object' && option.label !== undefined) ? option.label : option))
```

Comment:
I think this would be more readable if the ternary were on a separate line like it was before.

Reply:
Fixed